libs/newsletters-data-client/src/lib/user-profile.ts

```diff
@@ -56,5 +57,23 @@ export const levelToPermissions = (
 			UserAccessLevel.Drafter,
 		].includes(accessLevel),
 		viewMetaData: [UserAccessLevel.Developer].includes(accessLevel),
+		useJsonEditor: [UserAccessLevel.Developer].includes(accessLevel),
 	};
 };
+
+export const noPermissionMessage = (
```

Comment:
Not sure where this is called?  (I did a search but couldn't find anything, but it's always possible I did something silly)

Reply:
oops - I removed that, but didn't push the change!